### ISSUE TITLE
Unable to apply text colour to a selected text using the screen reader (Voice Over)

### ISSUE THREAD

**Issue by dufipl** [ckeditor/ckeditor4#5285]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open http://0.0.0.0:1030/tests/plugins/button/manual/menubuttonscreenreader and enable VoiceOver
2. Select the text
3. Try to apply colour to the selected text

### Expected result

Colour applied

### Actual result

Colour not applied, selection disappeared 

https://user-images.githubusercontent.com/32898551/178965135-204acc0c-4f51-4989-ab94-aa3633c405ac.mov

## Other details

* Browser: Safari Mobile
* OS: Mac OS Monterey
* CKEditor version: 4.19.1 (pre-release)
* Installed CKEditor plugins: full all

**Comment by Comandeer** [comment]
The test should be ignored on mobile. Does your iPad [announce itself as a desktop browser](https://github.com/ckeditor/ckeditor4/issues/5113#issuecomment-1086938399)?

**Comment by jacekbogdanski** [comment]
Closing due to https://github.com/ckeditor/ckeditor4/issues/5285#issuecomment-1184358332
